### PR TITLE
Remove shoov tests.

### PR DESCRIPTION
**Pull request by DavidKohav**
#393

### CODE REVIEW

**Comment by DavidKohav** [comment]
Done

**Comment by DavidKohav** [comment]
Please check

File: behat/composer.lock [hunk 1]
```diff
@@ -1,1571 +0,0 @@
-{
```
Comment (amitaibu) [inline behat/composer.lock]: this file should be kept

File: behat/features/blog_post.feature [hunk 2]
```diff
@@ -1,19 +0,0 @@
-Feature: Blog post
```
Comment (amitaibu) [inline behat/features/blog_post.feature]: this should not be deleted

Comment (amitaibu) [inline behat/features/blog_post.feature]: @DavidKohav ^^

**Comment by DavidKohav** [comment]
Fixed

**Comment by DavidKohav** [comment]
@amitaibu - ^^

**Comment by amitaibu** [comment]
Thanks.

**Pull request merged**